src/main/webapp/script.js

```diff
@@ -236,3 +251,16 @@ function getTooltipContent(node) {
 
   return content;
 }
+
+/**
+ * When a next/previous button is clicked, modifies the mutation index of the
+ * current graph to represent the new state. Then, the corresponding
+ * graph is requested from the server.
+ */
+function navigateGraph(amount) {
```

Comment:
what if the amount is greater than the number of mutations?

Reply:
I added a parameter in another branch (mutBackend) for this because there it would have a meaningful value. Do you think I should add it here anyways?

Reply:
I think it would be useful for the user to know when they're at the end! right now does the user know if they're at 0 or at the end? (maybe instead of the roots you could pass mutList.size() to the client?) and then display something when they're either at 0 or at end

Reply:
That's a good point. So right now, I'm going to add a dummy value to the JS for the maximum next clicks they can make. In the next PR I will actually populate this value from the backend :)